Ecommerce/EcommerceWeb/wwwroot/js/order.js

```
@@ -0,0 +1,50 @@
+                }
+                else {
+                    loadDataTable("all");
+                }
+            }
+        }
+    }
+});
+
+function loadDataTable(status) {
+    dataTable = $('#tblData').DataTable({
+        "ajax": { url: '/admin/order/getall?status=' + status },
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)